packages/core/src/Card/Card.stories.tsx

```diff
@@ -74,6 +88,27 @@ export const BoxShadowsWithVaryingBorderRadii = () => (
   </Box>
 )
 
+export const Default = () => (
+  <a id='PABLO' href='www.priceline.com'>
+    Pablo
+  </a>
+)
+Default.parameters = {
+  pseudo: {rootSelector: "PABLO" },
+}
```

Comment:
It looks like this is creating a story called `Default` that doesn't use Card. Is that intentional?

Reply:
we shouldn't leave ids with our names in the sourcecode, anyway